metaflow/task.py

```diff
@@ -92,6 +92,18 @@ def property_setter(
             setattr(cls, var, property(fget=lambda _, val=val: val, fset=set_cls_var))
             vars.append(var)
 
+        # We also passdown _graph_artifact through the entire graph
+        setattr(
```

Comment:
we have the `setattr(...property(fget=lambda ...` stanza repeated three times in this function. Maybe you can lift it to a nested function or something, since it is cryptic enough without repetition

Reply:
I can try but they are all different setattr. Let me see if I can make it a little less cryptic.